param/parameterized.py

```diff
@@ -2597,7 +2704,45 @@ class or instance that contains an iterable collection of
             for obj in sublist:
                 obj.param.set_dynamic_time_fn(time_fn,sublistattr)
 
-    def serialize_parameters(self_, subset=None, mode='json'):
+    def serialize_parameters(self_, subset=None, mode='json')->str:
+        """Returns the serialized parameters of the Parameterized object.
+
+        Parameters
+        ----------
+        subset (list, optional):
```

Comment:
Seems to be that's not how Numpy Docstrings are usually formatted? Based on https://numpydoc.readthedocs.io/en/latest/format.html#parameters.

```suggestion
        subset: list, optional
```